### PR TITLE
feat: remove legacy prop v0.47

### PR DESCRIPTION
**Pull request by aljo242**
Fixes QCK-513

### CODE REVIEW

File: x/interchainstaking/handler.go [hunk 1]
```diff
@@ -14,8 +14,10 @@ func NewProposalHandler(k *keeper.Keeper) govv1beta1.Handler {
 	return func(ctx sdk.Context, content govv1beta1.Content) error {
 		switch c := content.(type) {
 		case *types.RegisterZoneProposal:
+			// nolint: staticcheck // remove in v1.16
```
Comment (joe-bowman) [inline x/interchainstaking/handler.go:17]: v1.16? or v1.6?

File: x/interchainstaking/keeper/msg_server.go [hunk 2]
```diff
@@ -28,6 +35,313 @@ func NewMsgServerImpl(keeper Keeper) types.MsgServer {
...
+	if zone.IsSubzone() {
+		if err := types.ValidateSubzoneForBasezone(*zone, baseZone); err != nil {
+			return &types.MsgRegisterZoneResponse{}, err
+		}
+	}
+
+	k.SetZone(ctx, zone)
+
+	// generate deposit account
+	portOwner := chainID + ".deposit"
```
Comment (joe-bowman) [inline x/interchainstaking/keeper/msg_server.go:130]: Can we use the constants here please :)

https://github.com/ingenuity-build/quicksilver/blob/388cc8a5ea65a685d905c5790c5cd7f35303da69/x/interchainstaking/types/keys.go#L38

File: x/interchainstaking/keeper/msg_server.go [hunk 3]
```diff
@@ -28,6 +35,313 @@ func NewMsgServerImpl(keeper Keeper) types.MsgServer {
...
+			if tmClientState.Status(ctx, k.IBCKeeper.ClientKeeper.ClientStore(ctx, connection.ClientId), k.IBCKeeper.Codec()) != ibcexported.Active {
+				return &types.MsgUpdateZoneResponse{}, errors.New("new connection client state is not active")
+			}
+
+			zone.ConnectionId = change.Value
+
+			k.SetZone(ctx, &zone)
+
+			// generate deposit account
+			portOwner := zone.ID() + ".deposit"
```
Comment (joe-bowman) [inline x/interchainstaking/keeper/msg_server.go:304]: And consts here too please

Comment (aljo242) [reply x/interchainstaking/keeper/msg_server.go:304]: Will use https://github.com/ingenuity-build/quicksilver/pull/502/commits/f22910740f719ccde6c6a1c6d354a96bf6cab222 once #502 is merged

**Review by joe-bowman — APPROVED** [review]
lgtm

**Review by muku314115 — APPROVED** [review]
seems fine

**Review by ajansari95 — APPROVED** [review]
LGTM

**Review by muku314115 — APPROVED** [review]

**Review by ajansari95 — APPROVED** [review]
LGTM

**Review by muku314115 — APPROVED** [review]

**Pull request merged**